code/modules/mob/living/simple_animal/hostile/mining_mobs/mining_mobs.dm

```diff
@@ -59,3 +59,8 @@
 			visible_message(span_notice("The [T.name] [throw_message] [src.name]!"))
 			return
 	..()
+	
+/mob/living/simple_animal/hostile/asteroid/Life(delta_time = SSMOBS_DT, times_fired)
+	..()
+	if(is_station_level(z) && istype(get_area(src), /area/station))
+		adjustHealth(-2)
```

Comment:
Might want to do % of max_health. There's ones that have 40HP, and there's ones that have 1K HP.